test/e2e/tests/confirmations/transactions/erc20-token-send-redesign.spec.ts

```diff
@@ -89,30 +92,6 @@ async function mocks(server: Mockttp) {
   return [await mockedSourcifyTokenSend(server)];
 }
 
-export async function mockedSourcifyTokenSend(mockServer: Mockttp) {
```

Comment:
moved to the helpers file. We shouldn't export functions from a spec file to another, as this causes run all the specs in the imported file, duplicating runs and causing more time for the test run